### ISSUE TITLE
Adding "other" as an option for adduct

### ISSUE THREAD

**Issue by allegra-aron** [CCMS-UCSD/GNPS_Workflows#313]
When adding a spectrum for annotation, there is a list of adducts but this list is not complete. Maybe we can add "other" as an option for more rare adducts that we might want to annotate in GNPS...

**Comment by robinschmid** [comment]
In the MZmine direct submission of library spectra we already allow any kind of adduct

**Comment by mwang87** [comment]
We need to control this, so if you have adducts you want, you'll need to specifically list them.

**Comment by robinschmid** [comment]
This would be a very long list. But we could provide one. What do you think? @allegra-aron @mwang87

**Comment by allegra-aron** [comment]
yeah... for me, it would be useful to just add the M+Fe3+-2H+ adduct. I'm just trying to annotate siderophores that are iron-bound, in addition to the protonated adduct.
